### PR TITLE
Add support to store and messages builders

### PR DESCRIPTION
**Pull request by brunojdo**
#### What is the purpose of this pull request?
As the title says

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: store/blocks.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "address-locator": {
+    "blocks": true,
+    "inheritComponent": true
+  },
+  "address-rendeem": {
```
Comment (jgfidelis) [inline store/blocks.json]: `address-rendeem` é um typo? Seria `address-redeerm`?

File: store/interfaces.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "address-locator": {
+    "component": "Search"
+  },
+  "address-rendeem": {
```
Comment (jgfidelis) [inline store/interfaces.json]: typo tbm?

File: store/blocks.json (hunk 1 above)
Comment (lbebber) [reply store/blocks.json]: @jgfidelis fez um typo comentando sobre o typo hahah

File: store/interfaces.json (hunk 2 above)
Comment (lbebber) [inline store/interfaces.json]: aqui tambem

**Pull request merged**